src/parser/parser.yy

```diff
@@ -555,6 +555,15 @@ unreserved_keyword
     ;
 
 expression
+    : expression_internal {
+        $$ = $1;
+        if(!graph::ExpressionUtils::checkExprDepth($$)){
+            throw nebula::GraphParser::syntax_error(@1, "The above expression's depth exceeds the maximum depth!");
```

Comment:
need to delete this $$ before throw exception

Reply:
The above expression's depth exceeds the maximum depth: 512!

ExpressionUtils::kMaxDepth

Reply:
ok